### PR TITLE
create system users

### PR DESCRIPTION
**Pull request by WalkerTR**
Let Airlock create new ceph users as system users

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ before_script:
 - sbt ++$TRAVIS_SCALA_VERSION clean compile
 - echo "Wait for containers to be up and running"
 - bash waitForContainerSetup.sh
+- docker-compose exec ceph radosgw-admin user modify --uid ceph-admin --system
```
Comment (arempter) [inline .travis.yml]: I think we do not need this, as ceph-admin should be aready system user

File: README.md [hunk 2]
```diff
@@ -115,9 +112,9 @@ before diving in here. That will introduce you to the various components used.
    
             docker exec -it airlock_ceph_1 bash
             
-   3. Set the `testuser` to be a `system` user:
+   3. Set the `ceph-admin` to be a `system` user:
    
-            radosgw-admin user modify --uid testuser --system
+            radosgw-admin user modify --uid ceph-admin --system
```
Comment (arempter) [inline README.md]: Here I think step with testuser is ok.
This step is only required if we are not able to create system user by proxy. If we can we can remove whole point

File: README.md [hunk 3]
```diff
@@ -98,14 +98,11 @@ before diving in here. That will introduce you to the various components used.
     > everytime your session expires.
  
 4. Technically you're now able to use the aws cli to perform any commands through Airlock
-   to S3. Airlock automatically creates the user on Ceph for you. One thing it cannot do though, is make this user an 
-   admin/system user. Because of this, users on Ceph are not allowed to perform actions on other users' buckets.
+   to S3. Airlock automatically creates the user on Ceph for you. Since Ranger is in place to handle authorisation,
+   all users on Ceph can be allowed to do everything.
```
Comment (arempter) [inline README.md]: I would say maybe that:
all user authorization is handled on ranger level, so required permissions can be defined for users.
everything sounds not right from sec point of view

File: .travis.yml (hunk 1 above)
Comment (WalkerTR) [reply .travis.yml]: If you start a new container and you type `radosgw-admin user info --uid ceph-admin`, you can see that ceph-admin is actually not an admin (the `"system": true` entry is missing).

File: README.md (hunk 3 above)
Comment (WalkerTR) [reply README.md]: That sentence was already in the README (line 104).

Users in Ceph are indeed able to do everything (they all are admin), because authorization is provided by Ranger. I will rephrase it to make it clearer.

File: .travis.yml (hunk 1 above)
Comment (arempter) [reply .travis.yml]: ceph-admin user is beeing granted 
 radosgw-admin --cluster ceph caps add '--caps=buckets=*;users=*;usage=*;metadata=*' --uid=ceph-admin

on start, which is enough and we do not need to promote it to system.

File: README.md (hunk 2 above)
Comment (arempter) [reply README.md]: yes, airlock is creating users and before the change user has been created with no system permission, hence this step described as manual.
each user is created with name taken from request - not ceph-admin
